go_pprof_scraper/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "display_name": "Go-pprof-scraper",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
```

Comment:
You need to update your `ddev` command to the latest version. We do not support the v1 anymore, it should generate you a v2 file now. Alternatively you could use https://datadoghq.dev/integrations-core/ddev/cli/#ddev-meta-manifest-migrate